# Code : Square Pattern
Print the following pattern for the given N number of rows.
Pattern for N = 4
4444
4444
4444
4444


In [8]:
n = int(input())
i = 1
while i <= n :
    j = 1
    while j <= n:
        print(n,end='')
        j=j+1
    print()
    i=i+1
        

4
4444
4444
4444
4444


#  Triangular Star Pattern
Print the following pattern for the given N number of rows.
Pattern for N = 4

In [10]:
n = int(input())
i = 1
while i <= n :
    j = 1
    while j <= i:
        print('*',end='')
        j=j+1
    print()
    i=i+1

4
*
**
***
****


# Triangle Number Pattern
Print the following pattern for the given N number of rows.
Pattern for N = 4
1
22
333
4444
Input format :
Integer N (Total no. of rows)
Output format :
Pattern in N lines
Constraints
0 <= N <= 50
Sample Input 1:
5
Sample Output 1:
1
22
333
4444
55555
Sample Input 2:
6
Sample Output 2:
1
22
333
4444
55555
666666


In [12]:
n = int(input())
i = 1
while i <= n :
    j = 1
    while j <= i:
        print(j,end='')
        j=j+1
    print()
    i=i+1

6
1
12
123
1234
12345
123456


In [18]:
n = int(input())
i = 1
while i <= n :
    j = 1
    while j <= i:
        print(,end='')
        j=j+1
    print()
    i=i+1

4
0
01
012
0123


# Alpha Pattern
Print the following pattern for the given N number of rows.
Pattern for N = 3
 A
 BB
 CCC
Input format :
Integer N (Total no. of rows)
Output format :
Pattern in N lines
Constraints
0 <= N <= 26
Sample Input 1:
7
Sample Output 1:
A
BB
CCC
DDDD
EEEEE
FFFFFF
GGGGGGG
Sample Input 2:
6
Sample Output 2:
A
BB
CCC
DDDD
EEEEE
FFFFFF



In [22]:
n = int(input())
i = 1
while i <= n:
    j = 1
    start_char = chr(ord('A') + i - 1)
    while j <= i:
        # print jth column
         
        print(start_char, end='')
        j = j + 1
    print()
    i = i + 1

7
A
BB
CCC
DDDD
EEEEE
FFFFFF
GGGGGGG


# Character Pattern
Print the following pattern for the given N number of rows.
Pattern for N = 4
A
BC
CDE
DEFG
Input format :
Integer N (Total no. of rows)
Output format :
Pattern in N lines
Constraints
0 <= N <= 13
Sample Input 1:
5
Sample Output 1:
A
BC
CDE
DEFG
EFGHI
Sample Input 2:
6
Sample Output 2:
A
BC
CDE
DEFG
EFGHI
FGHIJK



In [25]:
n = int(input())
i = 1
while i <= n:
    j = 1
    start_char = chr(ord('A') + i - 1)
    while j <= i:
        # print jth column
        charP = chr(ord(start_char) + j - 1)
        print(charP, end='')
        j = j + 1
    print()
    i = i + 1

4
A
BC
CDE
DEFG


# Interesting Alphabets
Print the following pattern for the given number of rows.
Pattern for N = 5
E
DE
CDE
BCDE
ABCDE
Input format :
N (Total no. of rows)
Output format :
Pattern in N lines
Constraints
0 <= N <= 26
Sample Input 1:
8
Sample Output 1:
H
GH
FGH
EFGH
DEFGH
CDEFGH
BCDEFGH
ABCDEFGH
Sample Input 2:
7
Sample Output 2:
G
FG
EFG
DEFG
CDEFG
BCDEFG
ABCDEFG


In [27]:
n = int(input())
i = 1
while i <= n:
    j = 1
    start_char = chr(ord('E') - i + 1)
    while j <= i:
        # print jth column
        charP = chr(ord(start_char) + j - 1)
        print(charP, end='')
        j = j + 1
    print()
    i = i + 1

5
E
DE
CDE
BCDE
ABCDE


# Inverted Number Pattern
Print the following pattern for the given N number of rows.
Pattern for N = 4
4444
333
22
1
Input format :
Integer N (Total no. of rows)
Output format :
Pattern in N lines
Constraints :
0 <= N <= 50
Sample Input 1:
5
Sample Output 1:
55555 
4444
333
22
1
Sample Input 2:
6
Sample Output 2:
666666
55555 
4444
333
22
1


In [29]:
n = int(input())
i = 1
while i <= n:
    j = 1
    tsr_ = n - i + 1
    while j <= n - i + 1:
        print(tsr_, end="")
        j = j + 1
    print()
    i = i + 1

5
55555
4444
333
22
1


# Mirror Number Pattern
Print the following pattern for the given N number of rows.
Pattern for N = 4

 
The dots represent spaces.

Input format :
Integer N (Total no. of rows)
Output format :
Pattern in N lines
Constraints
0 <= N <= 50
Sample Input 1:
3
Sample Output 1:
      1 
    12
  123
Sample Input 2:
4
Sample Output 2:
      1 
    12
  123
1234



In [32]:
n = int(input())
i = 1
while i <= n:
    spaces = 1
    while spaces <= n - i:
        print(' ', end= "")
        spaces = spaces + 1
    j = 1
    while j <= i:
        print(j, end= "")
        j = j + 1
    print()
    i = i + 1

5
    1
   12
  123
 1234
12345


# Star Pattern
Print the following pattern
Pattern for N = 4

 
The dots represent spaces.

Input Format :
N (Total no. of rows)
Output Format :
Pattern in N lines
Constraints :
0 <= N <= 50
Sample Input 1 :
3
Sample Output 1 :
   *
  *** 
 *****
Sample Input 2 :
4
Sample Output 2 :
    *
   *** 
  *****
 *******


In [34]:
n = int(input())
i = 1
while i <= n:
    spaces = 1
    # spaces
    while spaces <= n - i:
        print(' ', end='')
        spaces = spaces + 1
    star = 1
    j = 1
    # increasing seq
    while j <= i:
        print('*', end='')
        j = j + 1
        star = star + 1
    # decreasing seq
    star = i - 1
    while star >= 1:
        print('*', end='')
        star = star -1
    print()
    i = i + 1

7
      *
     ***
    *****
   *******
  *********
 ***********
*************
